packages/react/src/types/Portal.ts

```diff
@@ -0,0 +1,8 @@
+export type PortalProps = {
+  /**
+   * Portals the floating element into a given container element — by default, outside of the app root and into the body.
```

Comment:
We aren't giving them the option to choose container element so suggest we alter the wording abit.
```suggestion
   * Portals the floating element outside of the app root and into the body.
```